repository/repo-operations.go

```diff
@@ -463,23 +536,23 @@ func openPullRequest(config *config.GitXargsConfig, repo *github.Repository, bra
 			switch {
 			case strings.Contains(err.Error(), "Draft pull requests are not supported"):
 				prErrorMessage = "Error opening pull request: draft PRs not supported for this repo. See https://docs.github.com/en/pull-requests/collaborating-with-pull-requests/proposing-changes-to-your-work-with-pull-requests/about-pull-requests#draft-pull-requests"
-				config.Stats.TrackSingle(stats.RepoDoesntSupportDraftPullRequestsErr, repo)
+				config.Stats.TrackSingle(stats.RepoDoesntSupportDraftPullRequestsErr, pr.Repo)
 
 			case strings.Contains(err.Error(), "Field:base Code:invalid"):
```

Comment:
Since this is such special logic, and it can change if the GitHub API changes, you could save some time later by making these strings ("Draft pull requests are not supported" etc) into `const`s and store them somewhere easy to change, like a new `constants.go` file.

Reply:
Great idea! I've filed this to track: https://github.com/gruntwork-io/git-xargs/issues/68